pkg/test/framework/components/apps/local/envoy/admin_util.go

```diff
@@ -149,7 +149,10 @@ func GetConfigDump(adminPort int) (*envoy_admin_v2alpha.ConfigDump, error) {
 // IsClusterPresent inspects the given Envoy config dump, looking for the given cluster
 func IsClusterPresent(cfg *envoy_admin_v2alpha.ConfigDump, clusterName string) bool {
 	clusters := envoy_admin_v2alpha.ClustersConfigDump{}
-	if err := clusters.Unmarshal(cfg.Configs["clusters"].Value); err != nil {
+	if len(cfg.Configs) < 3 {
```

Comment:
We should reuse the configdump wrapper in istioctl here or put it somewhere else and use it in both places

Reply:
Updated to use the configdump wrapper.